### ISSUE TITLE
The package doesn't work with Symfony 3.3.0+

### ISSUE THREAD

**Issue by mlocati** [concrete5/cloudflare_proxy#3]
This package [calls directly `Request::setTrustedProxies()`](https://github.com/concrete5/cloudflare_proxy/blob/875fec5aa4ed98b7f29f6130d139e9c2b9bdb4c8/src/CloudflareServiceProvider.php#L31), which requires a 2nd parameter since Symfony 3.3.0+.

https://github.com/concrete5/concrete5/pull/6711 fixes the `Request::setTrustedProxies()` call for the core, so IMHO the package should simply store its data to `concrete.security.trusted_proxies.ips` and let the core handle it.

If it's ok, I have some code that already implements it, and I can create a PR.

**Comment by aembler** [comment]
I agree, let's just make this package one that retrieves the IPs, and use the core mechanisms to set them.

**Comment by KorvinSzanto** [comment]
What @aembler describes is already what this package is. I think if we'd do this in a different way, we'd use `$config->set(...)` to overwrite the core settings at runtime similar to how we are doing it now.
